src/heroes/sombra/init.opy

```diff
@@ -12,35 +13,35 @@ def initSombra():
         # eventPlayer._self_healing_percent = 0.1 # set self healing to 0.1% such that 75hp healthpack becomes 0.075hp
         
     # setBaseDamage(eventPlayer, ADJ_SOMBRA_SMG_DAMAGE/OW2_SOMBRA_SMG_DAMAGE)
-
```

Comment:
remove random extra space

Reply:
I redid my commits to only touch one part of this file So this should be fixed